<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="gani" failures="1" tests="3" name="Test" time="54.307" errors="0" timestamp="2024-09-01T18:17:13 IST">
  <testcase classname="contacts.CreateContactWithLeadSource" name="createContactWithLeadSource" time="6.942"/>
  <testcase classname="contacts.CreateContactUsingCommonDataAndTestData" name="createContactUsingCommonDataAndTestData" time="6.653">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contacts.CreateContactUsingCommonDataAndTestData.createContactUsingCommonDataAndTestData(CreateContactUsingCommonDataAndTestData.java:32)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- createContactUsingCommonDataAndTestData -->
  <testcase classname="contacts.DeleteAllContacts" name="deleteAllContacts" time="5.488"/>
  <testcase classname="genericUtilities.BaseClass" name="@BeforeSuite dataBaseConnection" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite dataBaseConnection -->
  <testcase classname="genericUtilities.BaseClass" name="@BeforeSuite dataBaseConnection" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite dataBaseConnection -->
  <testcase classname="genericUtilities.BaseClass" name="@AfterSuite dataBaseConnectionClose" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite dataBaseConnectionClose -->
  <testcase classname="genericUtilities.BaseClass" name="@AfterSuite dataBaseConnectionClose" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite dataBaseConnectionClose -->
</testsuite> <!-- Test -->
